### PR TITLE
Damage Engine Fixed

### PR DESCRIPTION
**Pull request by HerlySQR**
Hello, I made the changes of the Damage Engine, but I'm not sure of how just not show all the commits, but I think now just show the last changes.

### CODE REVIEW

File: wurst/_handles/Destructable.wurst [hunk 1]
```diff
@@ -1,8 +1,11 @@
 package Destructable
 import NoWurst
-import Vectors
 import Wurstunit
 import Annotations
+import initlater Unit
```
Comment (Frotty) [inline wurst/_handles/Destructable.wurst:5]: don't add initlater import. Also this stuff does not look like it belongs in stdlib. Put it into some own package and then you also don't need initlater.

Comment (HerlySQR) [reply wurst/_handles/Destructable.wurst:5]: I could add a package called ElevatorWalls, but where could be good place it?

Comment (Frotty) [reply wurst/_handles/Destructable.wurst:5]: In your own project, not the stdlib.

Comment (HerlySQR) [reply wurst/_handles/Destructable.wurst:5]: ok

Comment (HerlySQR) [reply wurst/_handles/Destructable.wurst]: Very well, I removed it.

**Comment by Frotty** [comment]
Looking good now, thanks 👍

**Pull request merged**